clients/admin-ui/src/features/system/privacy-declarations/PrivacyDeclarationStep.tsx

```diff
@@ -15,6 +15,23 @@ const PrivacyDeclarationStep = ({ system }: Props) => {
     includeDatasets: true,
   });
 
+  const allEnabledDataCategories = dataProps.allDataCategories.filter(
```

Comment:
I think it might make more sense to do this filtering in `usePrivacyDeclarationData` instead:

https://github.com/ethyca/fides/blob/3710-enabled-toggle-in-taxonomy/clients/admin-ui/src/features/system/privacy-declarations/hooks.ts#L24

that's the "source" of the data, so I think it'd be best to transform it as early as we can